tests/microbenchmarks.py

```diff
@@ -66,11 +74,14 @@ def test_global_var_load(test_case):
             print(x/y)
             print(x)
             print(y)
-    """), timed=True)
+    """),
+        timed=True)
+
 
 def test_class_var_load(test_case):
     print("Running", "test_class_var_load")
-    test_case.runAsJava(adjust("""
+    test_case.runAsJava(
+        adjust("""
```

Comment:
At column 9: (F405) 'adjust' may be undefined, or defined from star imports: utils